magento/pwa-studio
# [Owls] PB-376: Video Background on PWA Studio (Venia) Storefront

## Description
As a Developer I want Row/Banner/Slide Pagebuilder PWA components to be able to render the video background that is configured by a Merchant in Admin Panel so that there is a feature parity between PageBuilder for traditional and PWA storefronts that saves my time on developing them

## Related Issue
[PB-376](https://jira.corp.magento.com/browse/PB-376) Video Background on PWA Studio (Venia) Storefront

## Acceptance 
 * New properties of content types are included in the components API (inline with our business orientated over developer properties best practice) and are successfully rendered inline with existing Luma experience in Venia PWA. Following the PWA Coding Standards.
 * The content types implement all required responsive behavior inline with current functionality provided in Luma theme.
### Verification Stakeholders

### Specification

### Verification Steps
1. Create Row/Banner/Slider with video background
3. Make sure Row/Banner/Slider contains video background.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
